.github/workflows/build.yml

```
@@ -198,70 +164,45 @@ jobs:
         run: pnpm install --frozen-lockfile
 
-      - name: Test
-        run: pnpm test:ci
-
-      - uses: actions/upload-artifact@v3
-        if: always()
-        with:
-          name: playwright-report
-          path: apps/wing-console/console/app/playwright-report/
-          retention-days: 30
+      - name: Install Playwright
+        working-directory: apps/wing-console/console/app
+        run: pnpm exec playwright install --with-deps
 
-      - name: Create git patch
-        id: diff
-        run: |
-          git add --all
-          git diff --staged --binary --patch > build.diff
-          if [ -s build.diff ]; then
-            echo "Diff found, creating a patch to apply later"
-            cat build.diff
-            echo "diff=true" >> $GITHUB_OUTPUT
-          fi
-      - name: Upload Diff
-        if: steps.diff.outputs.diff == 'true'
-        uses: actions/upload-artifact@v3
-        with:
-          name: build.diff
-          path: build.diff
+      - name: Test
+        run: pnpm test
 
-  benchmarks:
-    name: E2E Benchmarks
+  benchmark:
+    name: Benchmark
     runs-on: ubuntu-latest
     needs:
       - build
-    env:
-      HANGAR_WING_SPEC: "file:${{ github.workspace }}/wing/winglang-${{ needs.build.outputs.version }}.tgz"
-      HANGAR_WINGCONSOLE_APP_SPEC: "file:${{ github.workspace }}/wingconsoleapp/wingconsole-app-${{ needs.build.outputs.version }}.tgz"
-      HANGAR_WINGCONSOLE_SERVER_SPEC: "file:${{ github.workspace }}/wingconsoleserver/wingconsole-server-${{ needs.build.outputs.version }}.tgz"
-      HANGAR_WINGCONSOLE_DESIGN_SYSTEM_SPEC: "file:${{ github.workspace }}/wingconsoledesignsystem/wingconsole-design-system-${{ needs.build.outputs.version }}.tgz"
-      HANGAR_WINGCONSOLE_UI_SPEC: "file:${{ github.workspace }}/wingconsoleui/wingconsole-ui-${{ needs.build.outputs.version }}.tgz"
-      HANGAR_WINGCOMPILER_SPEC: "file:${{ github.workspace }}/wingcompiler/winglang-compiler-${{ needs.build.outputs.version }}.tgz"
-      HANGAR_WINGSDK_SPEC: "file:${{ github.workspace }}/wingsdk/winglang-sdk-${{ needs.build.outputs.version }}.tgz"
     steps:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Download Build Artifacts
-        uses: actions/download-artifact@v2
+      - name: Download Dist Artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: dist
+          path: dist
 
       - name: Setup pnpm
         uses: pnpm/action-setup@v2.2.4
         with:
           version: ${{ env.PNPM_VERSION }}
-        
+
       - name: Setup Node
         uses: actions/setup-node@v3
         with:
           cache: "pnpm"
           node-version: ${{ env.NODE_VERSION }}
 
       - name: Install Dependencies
-        run: pnpm install --ignore-scripts --frozen-lockfile --filter hangar --filter examples-valid --filter examples-invalid
+        run: pnpm install --frozen-lockfile --filter hangar --filter examples-valid --filter examples-invalid
```

scripts can be ignored here to avoid the extra wasi SDK setup

```suggestion
        run: pnpm install --ignore-scripts --frozen-lockfile --filter hangar --filter examples-valid --filter examples-invalid
```